### PR TITLE
Crazy8

### PR DESCRIPTION
**Pull request by MidnightBlackHorse17**

### CODE REVIEW

**Review by psk44 — APPROVED** [review]
The code looks good and everything seems to work! Good job!

File: source/CardGames.py [hunk 1]
```diff
@@ -36,6 +36,14 @@ def setValue(self):
     if self.value > 10:
       self.value = 10
     return self.value
+  
+  def eightCard(self):
+    if self.value == 8:
+      suit = input("What is the new suit? Options: Clubs, Hearts, Diamonds, Spades\n")
+      while suit not in {"Clubs", "Hearts", "Diamonds", "Spades"}:
+        print("Invalid input. Please enter new suit.")
```
Comment (Jaydenmd) [inline source/CardGames.py]: I think your code is really clean looking which is good! I was trying to do that for my code, but it is something that I need to work on. I think you have a fancy way of doing this task!

**Review by rastguelenianrn — COMMENTED** [review]
Everything looks good, but it looks like there was 2 HTML files in this commit (unless im just reading it wrong). Not really a problem i was just curious!

**Review by psk44 — COMMENTED** [review]
Good job creating the wild card! Now I want to play the game!

**Pull request merged**